POC/README.md

```diff
@@ -0,0 +1,39 @@
+# POC for asn1 - splitter
+
```

Comment:
please add an introduction section here, describing that this is a POC for splitting asn1 spec files, how it is developed and that it will be different from the main product